### ISSUE TITLE
Time Zone

### ISSUE THREAD

**Issue by dinosaw** [idleberg/Bootstrap-Listr#312]
After applying the changes mentioned here https://github.com/idleberg/Bootstrap-Listr/issues/225 (time stamp vs time ago), it changed what I needed to change - however the time zone appears to be off. It looks like it may be GMT. I'm looking to set the time zone to EST/EDT, or 'America/New_York'. Where exactly is the time zone handled for this application?

I've checked my php.ini and verified the correct time zone is set, as well as all my other PHP applications report the correct zone:
```
date/time support => enabled
"Olson" Timezone Database Version => 2015.5
Timezone Database => internal
Default timezone => America/New_York

Directive => Local Value => Master Value
date.default_latitude => 31.7667 => 31.7667
date.default_longitude => 35.2333 => 35.2333
date.sunrise_zenith => 90.583333 => 90.583333
date.sunset_zenith => 90.583333 => 90.583333
date.timezone => America/New_York => America/New_York
```
My server time is also set to America/New_York
```
[root@box ~]# date
Fri Apr 15 18:15:42 EDT 2016
```

**Comment by idleberg** [comment]
Thanks for the notice, a fix is now available in the `2.3-dev` branch

**Comment by dinosaw** [comment]
Awesome, thanks!
